js/navigation/NavigationLayout.js

```diff
@@ -1,22 +1,44 @@
-import React from "react";
-import ActivityScreen from "../screens/Activity";
-import CommunityScreen from "../screens/Community";
-import JournalScreen from "../screens/Journal";
-import DashBoardScreen from "../screens/Dashboard";
-import ActivityDetailsScreen from "../screens/ActivityDetails";
-import ActivityCompleteScreen from "../screens/ActivityComplete";
-import ProfileScreen from "../screens/Profile";
-import EditProfileScreen from "../screens/EditProfile";
-import CommunityCategoryScreen from "../screens/CommunityCategory";
-import CommunityQuestionScreen from "../screens/CommunityQuestion";
-import CommunityTopicScreen from "../screens/CommunityTopic";
-import { sharedNavigationOptions } from "./config";
+import React from 'react'
+import ActivityScreen from '../screens/Activity'
+import CommunityScreen from '../screens/Community'
+import JournalScreen from '../screens/Journal'
+import DashboardScreen from '../screens/Dashboard'
+import ActivityDetailsScreen from '../screens/ActivityDetails'
+import ActivityCompleteScreen from '../screens/ActivityComplete'
+import ProfileScreen from '../screens/Profile'
+import EditProfileScreen from '../screens/EditProfile'
+import CommunityCategoryScreen from '../screens/CommunityCategory'
+import CommunityQuestionScreen from '../screens/CommunityQuestion'
+import CommunityTopicScreen from '../screens/CommunityTopic'
+import AccountScreen from '../screens/Account'
+import SigninScreen from '../screens/UserSignin'
+import SignupScreen from '../screens/UserSignup'
+import ForgotPasswordScreen from '../screens/ForgotPassword'
+import ForgotPasswordConfirmationScreen from '../screens/ForgotPasswordConfirmation'
+import { sharedNavigationOptions } from './config'
 import {
   createStackNavigator,
   createBottomTabNavigator
-} from "react-navigation";
-import { Image } from "react-native";
-import styles from "./styles";
+} from 'react-navigation'
+import { Image } from 'react-native'
+import styles from './styles'
+
+export const AuthStack = createStackNavigator(
+  {
+    Signin: SigninScreen,
+    Signup: SignupScreen,
+    ForgotPassword: ForgotPasswordScreen,
+    ForgotPasswordConfirmation: ForgotPasswordConfirmationScreen
+  },
+  {
+    headerMode: 'none'
+  },
+  {
+    defaultNavigationOptions: ({ navigation }) => ({
+      ...sharedNavigationOptions(navigation)
+    })
+  }
+)
 
 const ProfileStack = createStackNavigator(
```

Comment:
this stack is not being used. Why? If you don't need it then remove it, please.

Reply:
devin's stack!

Reply:
devin removed it